app/views/articles/_show_main_content.html.erb

```diff
@@ -1,12 +1,16 @@
-    <%= render RecordToolbarComponent.new(presenter: document_presenter(@document), search_context: @search_context, search_session: search_session) %>
-    <% @page_title = t 'blacklight.search.show.title_html', :document_title => document_presenter(@document).html_title, :application_name => "#{I18n.t('blacklight.application_name')} catalog" -%>
-    <%# content_for(:head) { render_link_rel_alternates } -%>
- 
-    <%= render (blacklight_config.view_config(:show).document_component || Blacklight::DocumentComponent).new(document: document_presenter(@document), component: :div, title_component: :h1, show: true) do |component| %>
-      <% component.with_footer do %>
-        <% if @document.respond_to?(:export_as_openurl_ctx_kev) %>
-          <!-- COinS, for Zotero among others. -->
-          <span class="Z3988" title="<%= @document.export_as_openurl_ctx_kev(@document[@document.format_key]) %>"></span>
-        <% end %>
-      <%  end %>
+<div class="mt-4">
```

Comment:
Should this div only show conditionally?

Reply:
Oops. yes. Thanks